recipes/update_head_node_slurm.rb

```diff
@@ -16,17 +16,17 @@
 # limitations under the License.
 
 updated_cluster_config_path = "/tmp/cluster-config.updated.json"
-fetch_config_command = "#{node['cfncluster']['cookbook_virtualenv_path']}/bin/aws s3api get-object"\
-                       " --bucket #{node['cfncluster']['cluster_s3_bucket']}"\
-                       " --key #{node['cfncluster']['cluster_config_s3_key']}"\
+fetch_config_command = "#{node['cfncluster']['cookbook_virtualenv_path']}/bin/aws s3api get-object" \
+                       " --bucket #{node['cfncluster']['cluster_s3_bucket']}" \
+                       " --key #{node['cfncluster']['cluster_config_s3_key']}" \
                        " --region #{node['cfncluster']['cfn_region']} #{updated_cluster_config_path}"
 fetch_config_command += " --version-id #{node['cfncluster']['cluster_config_version']}" unless node['cfncluster']['cluster_config_version'].nil?
 shell_out!(fetch_config_command)
 if !File.exist?(node['cfncluster']['cluster_config_path']) || !FileUtils.identical?(updated_cluster_config_path, node['cfncluster']['cluster_config_path'])
   # Generate pcluster specific configs
   execute "generate_pcluster_slurm_configs" do
     command "#{node['cfncluster']['cookbook_virtualenv_path']}/bin/python #{node['cfncluster']['scripts_dir']}/slurm/pcluster_slurm_config_generator.py" \
-            " --output-directory /opt/slurm/etc/ --template-directory #{node['cfncluster']['scripts_dir']}/slurm/templates/ --input-file #{updated_cluster_config_path}"
+            " --output-directory #{node['cfncluster']['slurm']['install_dir']}/etc/ --template-directory #{node['cfncluster']['scripts_dir']}/slurm/templates/ --input-file #{updated_cluster_config_path}"
```

Comment:
This line is too long: Layout/LineLength: Line is too long. [203/200]
Can you break it into two lines with  `\`?
```
" --output-directory #{node['cfncluster']['slurm']['install_dir']}/etc/ --template-directory #{node['cfncluster']['scripts_dir']}/slurm/templates/ --input-file #{updated_cluster_config_path}"
```